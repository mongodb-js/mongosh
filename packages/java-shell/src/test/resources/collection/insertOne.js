// before
db.coll.remove({});
// command
const res = db.coll.insertOne({
    a: 1,
    objectId: new ObjectId('5ebaf064121756574a9767cf'),
    maxKey: new MaxKey(),
    minKey: new MinKey(),
    binData: new BinData(16, 'MTIzNA=='),
    date: new Date("2012-12-19"),
    isoDate: new ISODate("2012-12-19"),
    numberInt: NumberInt("24"),
    timestamp: new Timestamp(0, 100)
});
// command
res
// command
res.acknowledged
// command
db.coll.find();
// clear
db.coll.drop();
