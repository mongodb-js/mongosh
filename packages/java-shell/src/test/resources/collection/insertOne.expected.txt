{ "acknowleged": true, "insertedId": "UNKNOWN" }
[ { "_id": <ObjectID>, "a": 1, "objectId": <ObjectID>, "maxKey": { "$maxKey" : 1}, "minKey": { "$minKey" : 1}, "binData": { "$binary" : "MTIzNA==" , "$type" : 16}, "date": { "$date" : 1355875200000}, "isoDate": { "$date" : 1355875200000}, "numberInt": 24, "timestamp": Timestamp{value=429496729600, seconds=100, inc=0} } ]
